src/SDKs/Reservations/Management.Reservations/Generated/SdkInfo_AzureReservationAPI.cs

```diff
@@ -36,5 +36,5 @@ internal static partial class SdkInfo
       public static readonly String CodeGenerationErrors = "";
       public static readonly String GithubRepoName = "azure-rest-api-specs";
       // END: Code Generation Metadata Section  
-	  }
```

Comment:
This file should not be edited manually. Please regenerate the code. 
If there are issues with the generated file, please do the following
1. Pull latest changes from psSdkJson6
2. Run 
```
git clean -xdff
msbuild build.proj
```
3. Regenerate the code

Reply:
@dsgouda  Hi, 
I remembered this is a bug in the auto-generating script? Last time I had the same issue and had to fix this manually. :) 

Last time my PR : 
https://github.com/Azure/azure-sdk-for-net/pull/4725

Reply:
@luluRagdoll This has been fixed in our latest build tools. please follow the steps I mentioned above and regenerate the code

Reply:
@luluRagdoll 
Please follow the steps described above and regenerate the code

Reply:
@luluRagdoll Please follow the steps described above and regenerate the code